lib/cli_logging.dart

```diff
@@ -4,12 +4,11 @@
 
 /// This library contains functionality to help command-line utilities to easily
 /// create aesthetic output.
+library cli_logging;
```

Comment:
I don't really use them anymore either. Without the library directive however, when you sort imports in this file, the analysis server copies part of the dartdoc comment to after the import directives :(

(have not yet filed a bug)